challenge_3/python/returnlove/src/majority_element.py

```diff
@@ -0,0 +1,28 @@
+"""
+To find the majority elements in a list.
+	Counts number of times each element is present in the list
+	prints all the elements which are present more than half of the list
+"""
+
+import math
+
+def find_majority_element(input_list):
+	majority_condition = math.floor(len(input_list)/2)
+	print(majority_condition)	
+	my_dict = {}
+	for element in input_list:
+		if element not in my_dict.keys():
+			my_dict[element] = 1
+		else:
+			my_dict[element] += 1
+
+	for k in my_dict.keys():
+		if my_dict[k] > majority_condition:
+			print(k)
```

Comment:
@hkl0902 my bad, I forgot to remove that print statement. Re, returning the value, I didn't get your point. Even if we return a value in a function, we need to use print statement when calling that function. Nonetheless, I created a list where I store all the matching numbers and return the list. Let me know what you think. (I'm sure using of return statement inside the for loop creates a problem as only first attempt will get returned and it ignores the second match if any. At lease in JS).